bigflow/workflow.py

```diff
@@ -29,8 +30,36 @@ def daily_start_time(start_time: dt.datetime) -> dt.datetime:
     return start_time.replace(hour=0, minute=0, second=0, microsecond=0) - td
 
 
+class JobContext(typing.NamedTuple):
+    runtime_raw: typing.Optional[str]
+    runtime: typing.Optional[dt.datetime]
+    env: typing.Optional[str]
```

Comment:
Do we really need `env` in a job context? Up to this point, env was only known for the `Configuration` class, and the CLI, which only sets an environmental variable.

Reply:
No, it was committed by mistake. Removed.